documentation/working-with-testcafe-studio/running-tests.md

```diff
@@ -18,15 +18,15 @@ TestCafe Studio can run tests in all desktop and mobile browsers (with or withou
 
 ![Run Congiruation Menu](../../images/working-with-testcafe-studio/run-configuration-list.png)
 
-To modify configurations or create a new one, click the ![Settings button](../../images/working-with-testcafe-studio/settings-icon.png) button and use the [Run Configurations](../user-interface/run-configurations-dialog.md) dialog. You can select one or multiple browsers and specify options that affect test execution for each configuration.
+To modify configurations or create a new one, click the ![Settings button](../../images/working-with-testcafe-studio/settings-icon.png) button and use the [Run Configurations](../user-interface/run-configurations-dialog.md) dialog. You can select one or multiple browsers and specify their options.
```

Comment:
@dirk-pieterse 
> their options

their options = browsers options? no, options that affect test execution.

> For each configuration 

it's also important, since dialog displays several configurations, and you can modify settings for each configuration.

Reply:
No, for each configuration, you can select one or multiple browsers and specify options that affect test execution.

Reply:
Ok